packages/@react-spectrum/color/chromatic/ColorField.chromatic.tsx

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {ColorField} from '../';
+import {Grid, repeat} from '@react-spectrum/layout';
+import {mergeProps} from '@react-aria/utils';
+import {Meta, Story} from '@storybook/react';
+import React from 'react';
+import {SpectrumColorFieldProps} from '@react-types/color';
+
+// Ignore read only because it doesn't apply any distingishable visual features
+let states = [
+  {isQuiet: true},
+  {isDisabled: true},
+  {validationState: ['valid', 'invalid']},
+  {isRequired: true},
+  {necessityIndicator: 'label'}
+];
+
+// Generate a powerset of the options
+let combinations: any[] = [{}];
```

Comment:
We should really pull this out into a helper function, if you have the chance

Reply:
good idea, I'll see if I can do that.